secure_consul_acl/step5.md

```diff
@@ -15,7 +15,7 @@ service_prefix "" {
 }
 ```
 
-This policy will permit the registration of all nodes with a name starting with `client-`. This permits you to reuse a single token across all your clients or to combine this policy with more specific ones for the single client agent to generate specific tokens.
+This policy will permit the registration of all nodes with a name starting with `client-`. This permits you to reuse a single token across all your clients, or to combine this policy with more specific ones for the single client agent to generate specific tokens.
```

Comment:
I don't know what to suggest for this second sentence, but it's meaning is unclear to me.

Reply:
I wanted to say that you can use this policy as a base for all client agents (because `node_prefix:write` is valid only for nodes called client-*) by giving them all the same token or if needed combine this with other policies that might give additional permission such as kv store write or service write to permit you to register some services by creating other tokens (i.e. one per client).

Ideally we can even stop after the first sentence.